src/main/scala/sbtprojectgraph/Node.scala

```diff
@@ -3,23 +3,29 @@ package sbtprojectgraph
 import sbt.{ ProjectRef, ResolvedProject }
 
 /** A node in a dependency tree of elements of type `A`. */
-final case class Node[A](value: A, directDeps: Set[Dependency[Node[A]]], allDeps: Set[Dependency[A]], allEdges: Set[Edge[A]])
```

Comment:
`directDeps` variable was shadowed by `val directDeps` definition below (the non-shadowed value is not used anywhere before)